src/modules/tests/tests.actions.ts

```diff
@@ -89,7 +90,7 @@ export class SendCurrentTestSuccess implements Action {
 
 export class SendCurrentTestFailure implements Action {
   readonly type = SEND_CURRENT_TEST_FAILURE;
-  constructor(public slotId: string) {}
+  constructor(public slotId: string, public error: HttpErrorResponse) { }
```

Comment:
Do we need to send around the `slotId` considering that we are uploading the **current test**?

Reply:
Yeah, I would just need to ensure that in the test effects I get the current test. Hmmm. I will make this change.